src/components/action-bar/action-bar.tsx

```
@@ -170,10 +181,32 @@
+        if (!this.isExpandable) {
+            return;
+        }
+
+        return (
+            <button
+                class={{
+                    'expand-shrink': true,
+                }}
+                onClick={this.handleCollapseExpandClick}
+            >
+                <limel-icon name="double_left" />
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)